internal/pkg/deploy/cloudformation/stack/testdata/workloads/rdws-test.stack.yml

```diff
@@ -209,6 +209,7 @@ Resources:
       Workload: !Ref WorkloadName
       EnvStack: !Sub '${AppName}-${EnvName}'
       Parameters: [NATWorkloads,]
+      EnvVersion:
```

Comment:
nit: do you think we should add a mock `EnvVersion` to the runtime configuration so that it renders in the integration test instead of being left blank?